src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
```

Comment:
So we'll need a Rating model with name (which will be a fk to KitReview.name or KitReviewProperty.name) and the actual rating then which will serve as a through model? 

In case general rating to be determined based on the average of prop ratings , that would make them required. What if some users are lazy and want only to give general rating and not to rate each prop separately?